src/electionguard/ballot.py

```diff
@@ -596,6 +608,7 @@ def make_ciphertext_ballot_contest(
         description_hash=description_hash,
         ballot_selections=ballot_selections,
         nonce=nonce,
+        ciphertext=elgamal_accumulation,
```

Comment:
This highlights what I'm saying. I feel like this should be a much more meaningful line.